File: gists/start/clients/native-clients/dtos.swift
// @Route("/hello")
// @Route("/hello/{Name}")
public class Hello : IReturn, Codable
{
    public typealias Return = 
        HelloResponse

    public var name:String?

    required public init(){}
}

public class HelloResponse : Codable
{
    public var result:String?

    required public init(){}
}

File: gists/start/clients/web-client/dtos.ts
export class HelloResponse
{
    public result: string;

    public constructor(init?: Partial<HelloResponse>) { 
        (Object as any).assign(this, init); 
    }
}

// @Route("/hello")
// @Route("/hello/{Name}")
export class Hello implements IReturn<HelloResponse>
{
    public name: string;

    public constructor(init?: Partial<Hello>) { 
        (Object as any).assign(this, init); 
    }
    public createResponse() { return new HelloResponse(); }
    public getTypeName() { return 'Hello'; }
}